type_wrapper.py

```diff
@@ -0,0 +1,37 @@
+class Type(object):
+    pass
```

Comment:
I am wondering about naming, which is always subjective, but to me this class should really be called ``Value``, because instances of it are values not types.